### ISSUE TITLE
[2.5][2.0.181121-rc1] Integration Testing - Quick Deposit Button Missing

### ISSUE THREAD

**Issue by toi16** [bitshares/bitshares-ui#2180]
**Describe the bug**
Test https://wallet.bitshares.org/#/market/BTS_OPEN.ETH the "Deposit ETH" buttons in the exchange.  Are there two options to "Deposit ETH"?

**To Reproduce**
1. Go to 'https://wallet.bitshares.org/#/market/BTS_OPEN.ETH '
ERROR: Only one 'QUICK DEPOSIT' button is present (Openledger) when buying ETH..
**NOTE: In Chrome two buttons are shown**

**Expected behavior**
Two quick deposit buttons should be present when buying ETH.

**Screenshots**

![firefox deposits](https://user-images.githubusercontent.com/23018324/48624775-33a30f00-e9a5-11e8-8082-f3bfc79fb8ea.PNG)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser Safari
 - Version 63.0.3 (64-bit)

**Additional context**

BitShares.181031

**Comment by sschiessl-bcp** [comment]
My link was misleading ... apologies.

Please try https://staging.bitshares.org/#/market/BTS_OPEN.ETH

**Comment by toi16** [comment]
Firefox:
![firefox eth](https://user-images.githubusercontent.com/23018324/48628672-af09be00-e9af-11e8-833c-a86427ad3adb.PNG)

Only one button present.
Initially both 'DEPOSIT ETH' and 'DEPOSIT BTS' buttons were disabled and I had to do a screen refresh before they became enabled.
Tooltips initially did not work, however after a refresh they did.

**Comment by sschiessl-bcp** [comment]
If you click on "Deposit ETH" a popup with two choices should appear. That is what I meant and failed to describe properly. Can you please try?

**Comment by toi16** [comment]
**Chrome:**
![chrome eth](https://user-images.githubusercontent.com/23018324/48717590-a7495400-ec11-11e8-97f8-be7c414f47a6.PNG)
'Deposit ETH' does pop up with two choices correctly.

**Firefox:**
![ff eth](https://user-images.githubusercontent.com/23018324/48717666-c811a980-ec11-11e8-91de-b768be225ce2.PNG)
When you click on 'DEPOSIT ETH' the modal above appears. There is NO pop up with two choices.

**Comment by sschiessl-bcp** [comment]
That is both on staging wallet? Wow, totally weird

**Comment by toi16** [comment]
Yes they are both using the same URL (https://staging.bitshares.org/#/market/BTS_OPEN.ETH)

**Comment by sschiessl-bcp** [comment]
What version does the UI report in both cases in the bottom left? Should be the RC number

**Comment by toi16** [comment]
If it helps the tooltip in Firefox is:
data-tip="Gateways provide IOUs on the BitShares Blockchain that are backed by the real asset" 

The tooltip in chrome is: 
data-tip="Deposit ETH, click to open options"

**Comment by toi16** [comment]
UI Versions:
Chrome: bitshares.181121-rc1
Firefox: BitShares.181121-rc1

**Comment by startailcoon** [comment]
Your Chrome browser may have issues fetching data that relates to the Gateway functionality. If the UI doesn't think that there is any Gateway related assets to use, it will not give you the option to it either. We may need to check on the underlaying fetching code for this.

**Comment by sschiessl-bcp** [comment]
This needs investigation, but might simply be due to a short network failure when pinging.

@toi16 could you please open it again and simply reload it a couple of times to see if it consistently only displays one Button? And if it does display one button, can you please post the console error log?

**Comment by sschiessl-bcp** [comment]
Thank you very much, I appreciate your responsiveness!

@startailcoon I can reproduce this reliably with browserstack (Win10, Firefox 63), could you have a look at this issue? It might be that this option simply never worked so far in Firefox ...

**Comment by sschiessl-bcp** [comment]
PR did not fix

**Comment by startailcoon** [comment]
This would require some more checking then. I'm a bit confused to why it didn't work.

**Comment by startailcoon** [comment]
Will dig into this this sprint to sort out this and some other gateway specific issues.
